spec/src/main/asciidoc/ch03-entity-operations.adoc

```diff
@@ -2507,24 +2509,20 @@ event callback methods of the corresponding types.
 
 The following rules apply to lifecycle callback methods:
 
-* Lifecycle callback methods may throw
-unchecked/runtime exceptions. A runtime exception thrown by a callback
-method that executes within a transaction causes that transaction to be
-marked for rollback if the persistence context is joined to the
-transaction.
-* Lifecycle callbacks can invoke JNDI, JDBC,
-JMS, and enterprise beans.
-* In general, the lifecycle method of a
-portable application should not invoke _EntityManager_ or query
-operations, access other entity instances, or modify relationships
-within the same persistence
-contextfootnote:[Note that this
-caution applies also to the actions of objects that might be injected
-into an entity listener]footnote:[The semantics of such
-operations may be standardized in a future release of this
-specification.]. A lifecycle
-callback method may modify the non-relationship state of the entity on
-which it is invoked.
+* Lifecycle callback methods may throw unchecked/runtime exceptions.
+  A runtime exception thrown by a callback method that executes within
+  a transaction causes that transaction to be marked for rollback if
+  the persistence context is joined to the transaction.
+* Lifecycle callbacks can invoke JNDI, JDBC, JMS, and enterprise beans.
+* A lifecycle callback method may modify the non-relationship state of
+  the entity on which it is invoked.
+* In general, the lifecycle method of a portable application should not
+  invoke _EntityManager_ or query operations, access other entity
+  instances, or modify relationships within the same persistence
+  contextfootnote:[Note that this caution applies also to the actions of
+  objects that might be injected into an entity listener]footnote:[The
+  semantics of such operations may be standardized in a future release of
+  this specification.].
```

Comment:
was the removal of the last sentence (*) here intentional? I think it's better to be explicit over staying silent here

* _A lifecycle callback method may modify the non-relationship state of the entity on which it is invoked._

Reply:
I didn't remove it, I moved it to its own bullet point tight above.

Reply:
By the way, I would remove the footnote:

> The semantics of such operations may be standardized in a future release of this specification.

That's been there since JPA 1.0, and 17 years later: we've gone through several major versions, we have not standardized this, we have no plan to do so, and I think it would be _extremely_ hard to do so.

Reply:
I've missed that move, sorry. And I agree with the removal of that footnote as, in the end, it won't change the actual meaning anyway :-)